hazelcast-hibernate52/src/main/java/com/hazelcast/hibernate/CacheEnvironment.java

```diff
@@ -149,16 +147,12 @@ public static int getLockTimeoutInMillis(final Properties props) {
             Logger.getLogger(CacheEnvironment.class).finest(e);
         }
         if (timeout < 0) {
-            timeout = MAXIMUM_LOCK_TIMEOUT;
+            timeout = (int) MAXIMUM_LOCK_TIMEOUT.toMillis();
         }
         return timeout;
     }
 
     public static boolean shutdownOnStop(final Properties props, final boolean defaultValue) {
         return ConfigurationHelper.getBoolean(CacheEnvironment.SHUTDOWN_ON_STOP, props, defaultValue);
     }
-
-    public static boolean isExplicitVersionCheckEnabled(final Properties props) {
```

Comment:
Does `EXPLICIT_VERSION_CHECK`  property have any effect at all?  It's only set in test properties and never read.

Reply:
Will double check

Reply:
No, it's not used at all